contrib/endpoints/src/api_manager/service_control/aggregated_test.cc

```diff
@@ -321,8 +321,11 @@ TEST_F(QuotaAllocationTestWithRealClient, AllocateQuotaFailedTest) {
   info.metric_cost_vector = &metric_cost_vector_;
 
   FillOperationInfo(&info);
-  sc_lib_->Quota(info, nullptr, [](Status status) {
-    ASSERT_TRUE(status.code() == Code::RESOURCE_EXHAUSTED);
+  sc_lib_->Quota(info, nullptr, [info, this](Status status) {
```

Comment:
Please add comment.
Quota cache always allows the first call. The negative will take effect
only after remote server call is replied.  In this case, it will be the second call.